src/boosting/gbdt_model_text.cpp

```diff
@@ -393,7 +393,7 @@ std::string GBDT::SaveModelToString(int start_iteration, int num_iteration, int
     ss << loaded_parameter_ << "\n";
     ss << "end of parameters" << '\n';
   }
-  return std::move(ss.str());
+  return ss.str();
```

Comment:
/gha run r-valgrind

Reply:
Valgrind is happy: https://github.com/microsoft/LightGBM/actions/runs/323142426.

Reply:
if valgrind is happy, I am happy 😀